package.json

```diff
@@ -18,7 +18,24 @@
     "@angular/core": "^9.1.5"
   },
   "dependencies": {
```

Comment:
You should not have anything listed in the `dependencies` section. Move these to `devDependencies`?